index.md

```diff
@@ -15,7 +15,7 @@ plagued reflection-based solutions. More details can be found in
 
 ## Where are the docs?
 
-* [User documentation](users-guide.md)
+* [User documentation](users-guide.html)
```

Comment:
Can you just omit the file extension? Jekyll appears to handle this: http://google.github.io/dagger/users-guide